ibm/service/power/resource_ibm_pi_cloud_connection.go

```diff
@@ -224,10 +229,24 @@ func resourceIBMPICloudConnectionCreate(ctx context.Context, d *schema.ResourceD
 	}
 
 	client := st.NewIBMPICloudConnectionClient(ctx, sess, cloudInstanceID)
+	retryCount := 2
 	cloudConnection, cloudConnectionJob, err := client.Create(body)
 	if err != nil {
-		log.Printf("[DEBUG] create cloud connection failed %v", err)
-		return diag.FromErr(err)
+		for count := retryCount; count >= 0; count-- {
+			if err == nil {
```

Comment:
What is the use of this check? We are checking err is not nil before loop and also inside loop it is checking and returning.